meta-refkit/conf/distro/include/refkit-supported-recipes.txt

```diff
@@ -161,6 +161,11 @@ i2c-quark-board@quark-bsp
 icu@core
 iftop@networking-layer
 image-transport@ros-layer
+industrial-msgs@ros-layer
+industrial-robot-client@ros-layer
```

Comment:
It might make sense to merge this patch with the previous one that was changing the same file.